crates/pink/pink-extension/src/chain_extension.rs

```diff
@@ -199,6 +199,13 @@ pub trait PinkExt {
     /// Get the version of the current contract runtime in this cluster.
     #[ink(extension = 21, handle_status = false)]
     fn runtime_version() -> (u32, u32);
+
+    /// Batch http request
+    #[ink(extension = 22, handle_status = true)]
+    fn batch_http_request(
```

Comment:
Should we give out error on the ink side when there are too many requests?

Reply:
Given